glide-core/src/client/value_conversion.rs

```diff
@@ -1387,6 +1387,8 @@ fn convert_flat_array_to_array_of_pairs(
     Ok(Value::Array(result))
 }
 
+const IS_ARRAY: &'static (dyn Fn(Value) -> bool + Sync) = &|val| matches!(val, Value::Array(_));
```

Comment:
Equivalently, this could just be a top-level function:
```
fn is_array(val: Value) -> bool {
    matches!(val, Value::Array(_))
}
```
Then you could do `Some(&is_array)` instead. Up to you if you want to make this change though.